### DISCUSSION TITLE
Mamba f722 Flight Controller sda scl

### DISCUSSION THREAD

**Discussion by kiri233** [iNavFlight/inav#6432] [General]
There sda scl interfaces on the new version, consider whether support.

MAMBA F722S F7 Betaflight Flight Controller OSD 5/9V 2A BEC & F50PRO 50A 3-6S Blheli_32  Dshot1200 FPV Racing Brushless ESC for RC Drone
https://banggood.app.link/ZTURBuuVicb

**Comment by DzikuVx** [comment]
From what I know this board is supported as `MambaF722` target already

**Reply by kiri233** [reply]
mamba f722 There are several models ,The current version cannot support the sda scl interface, and cannot connect the magnetometer and barometer.

**Reply by djc0ntra** [reply]
I can confirm this also affects the Mamba F722APP flight controller.
